.chloggen/remove-component-status-ready.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Do we need a changelog for this? It looks like `status.Reporter` is internal so that API was not public

Reply:
I'll remove it. It's a completely internal refactor.